arch/M68K/M68KInstPrinter.c

```diff
@@ -338,6 +339,9 @@ const char* M68K_reg_name(csh handle, unsigned int reg)
 #ifdef CAPSTONE_DIET
 	return NULL;
 #else
+	if (reg >= S_REG_NAMES_COUNT) {
```

Comment:
instead of hardcoding, how about using ARR_SIZE marcro then?

so you can write this line like: `if (reg >= ARR_SIZE(s_reg_names) {`

Reply:
Yes I agree.